ports/cc3200/simplelink/oslib/osi.h

```diff
@@ -3,35 +3,35 @@
 //
 // MACRO and Function prototypes for TI-RTOS and Free-RTOS API calls
 //
-// Copyright (C) 2014 Texas Instruments Incorporated - http://www.ti.com/ 
-// 
```

Comment:
This simplelink is third-party code.  But I guess if we are going to add CI to check for trailing spaces, then probably it's easier to just remove these as well.

Reply:
> if we are going to add CI to check for trailing 

Isn't that already done. I have seen some complaints of the code formatting checker about trailing space.

Reply:
If it's third party code, then `ports/cc3200/simplelink/oslib/osi.h` is unlikely to be changed outside of total replacement? Thus it wont trigger the whitespace checks. Is it best left untouched?

afaik the only whitespace enforcement is via uncrustify, and just running that across the whole codebase results in yet another *huge* list of small code consistency changes- https://github.com/micropython/micropython/commit/18d0c7f0edcd7d7eaedfae0e762e8959af00fade

Reply:
> and just running that across the whole codebase results in yet another _huge_ list of small code consistency changes

You must be running it wrong, because the macro indent/dedent changes should not be there.  You need to run `tools/codeformat.py`.

Reply:
I was running:

```
find . -name *.c -not -path "./lib/*" -not -path "./ports/cc3200/FreeRTOS/*" -not -path "./drivers/cc3100/*" | xargs uncrustify -c tools/uncrustify.cfg -lC --replace --no-backup
```
But on closer inspection it looks like `tools/codeformat.py` has code (`fixup_c() `) to then *undo* these erroneous changes.

I tried running codeformat *after* directly running uncrustify and received a much smaller set of changes:

https://github.com/pimoroni/micropython/commit/052e0ad1a3a52923d582f95df9b63d5247811d15

Weirdly if I just run something like:

```
find . -name *.c -not -path "./lib/*" -not -path "./ports/cc3200/FreeRTOS/*" -not -path "./drivers/cc3100/*" | ./tools/codeformat.py -v -c -f
```

I get a long list of files that have ostensibly been considered and changed, but no actual changes... it was always my assumption that `codeformat.py` only picked up files changed in the last commit, but I can't actually find a mechanism for that.

Poking a little further, it looks like all of my changes are to files that are *NOT* in `PATHS` and not in `EXCLUSIONS˘, eg:  `drivers/bus/softspi.c` or `examples/natmod/btree/btree_c.c`.

I should probably raise another PR to consider these...

Reply:
PR raised here, since there are a few minor but salient issues that shouldn't be lost in this nested chat - https://github.com/micropython/micropython/pull/14046